lib/pf/UnifiedApi/Controller/Config.pm

```diff
@@ -92,6 +93,12 @@ sub build_list_search_info {
     return 200, $info;
 }
 
+=head2 items
+
+items
```

Comment:
In all the file the perldoc description is just the name of the sub

I know some of them are self-explanatory with their name but a few words to describe the less obvious ones would be welcome